src/task.c

```diff
@@ -1596,6 +1607,17 @@ jl_task_t *jl_init_root_task(jl_ptls_t ptls, void *stack_lo, void *stack_hi)
     ct->ptls = ptls;
     ct->world_age = 1; // OK to run Julia code on this task
     ct->reentrant_timing = 0;
+    ct->cpu_time_ns = 0;
+    ct->wall_time_ns = 0;
+    if (ct->is_timing_enabled) {
+        uint64_t now = jl_hrtime();
+        ct->first_enqueued_at = now;
+        ct->last_dequeued_at = now;
+    }
+    else {
+        ct->first_enqueued_at = 0;
+        ct->last_dequeued_at = 0;
+    }
```

Comment:
Should these also have 
```
# created -task-started-> user_time
```
edges?

Or do the root tasks go to wait_time or straight to user_time? I guess based on setting `last_dequeued_at` they go straight to user_time?